MAINTAINERS.yml

```
@@ -0,0 +1,1184 @@
+    status: maintained
+    maintainers: anangl
+    files:
+        - doc/reference/audio/i2s.rst
+        - drivers/i2s/
+        - dts/bindings/i2s/
+        - include/drivers/i2s.h
+        - tests/drivers/i2s/
+    labels:
+        - "area: I2S"
+
+"Drivers: IEEE 802.15.4":
+    status: maintained
+    maintainers:
+        - tbursztyka
+    collaborators:
+        - jukkar
+    files:
+        - drivers/ieee802154/
+
+"Drivers: Interrupt controllers":
+    status: orphaned
+    files:
+        - drivers/interrupt_controller/
+        - dts/bindings/interrupt-controller/
+        - include/drivers/interrupt_controller/
+        - include/dt-bindings/interrupt-controller/
+
+"Drivers: IPM":
+    status: maintained
+    maintainers:
+        - andrewboie
+    files:
+        - drivers/ipm/
+    description: >-
+        Inter-processor mailboxes
+
+"Drivers: kscan":
+    status: orphaned
+    files:
+        - drivers/kscan/
+        - include/drivers/kscan.h
+        - samples/drivers/espi/
+        - tests/drivers/kscan/
+
+"Drivers: LED":
+    # Provisional entry, subject to approval
+    status: maintained
+    maintainers:
+        - Mani-Sadhasivam
+    files:
+        - drivers/led/
+        - include/drivers/led/
+        - include/drivers/led.h
+        - samples/drivers/led_*/
+    labels:
+        - "area: LED"
+
+"Drivers: LED Strip":
+    status: maintained
+    maintainers:
+        - mbolivar-nordic
+    files:
+        - drivers/led_strip/
+        - dts/bindings/led_strip/
+        - include/drivers/led_strip.h
+
+"Drivers: lora":
+    # Provisional entry, subject to approval
+    status: maintained
+    maintainers:
+        - Mani-Sadhasivam
+    files:
+        - drivers/lora/
+        - include/drivers/lora.h
+        - samples/drivers/lora/
+
+"Drivers: Modem":
+    status: maintained
+    maintainers:
+        - mike-scott
+    files:
+        - drivers/modem/
+    labels:
+        - "area: Modem"
+
+"Drivers: Neural networking":
+    status: orphaned
+    files:
+        - drivers/neural_net/
+
+"Drivers: PCI":
+    status: maintained
+    maintainers:
+        - dcpleung
+    collaborators:
+        - andrewboie
+    files:
+        - drivers/pcie/
+        - include/drivers/pcie/
+    labels:
+        - "area: PCI"
+
+"Drivers: peci":
+    status: orphaned
+    files:
+        - drivers/peci/
+        - include/drivers/peci.h
+        - samples/drivers/peci/
+
+"Drivers: Pinmux":
+    status: maintained
+    maintainers:
+        - mnkp
+    files:
+        - doc/reference/peripherals/pinmux.rst
+        - drivers/pinmux/
+        - include/drivers/pinmux.h
+    labels:
+        - "area: Pinmux"
+
+"Drivers: PTP Clock":
+    # Provisional entry, subject to approval
+    status: maintained
+    maintainers:
+        - jukkar
+    files:
+        - drivers/ptp_clock/
+        - include/ptp_clock.h
+
+"Drivers: PWM":
+    # Provisional entry, subject to approval
+    status: maintained
+    maintainers: anangl
```

Should it be rather
```suggestion
    maintainers: 
        - anangl
```
?

---

Yes, thanks. Fixed now. Also similar occurrences of this problem.